api/tests/Behat/features-v2/reporting/sections/additional-information/additional-information.ndr.feature

```diff
@@ -0,0 +1,17 @@
+@v2 @additional-information
+Feature: Additional Information (NDR)
+
+@ndr-not-started
```

Comment:
Eye twitch moment but these tags could do with being indented to sit above each scenario (and in the other feature files too)